docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -51,6 +51,17 @@ are also available. They are discussed later in this topic.
   - Enable caching for partition metadata. You can disable caching to avoid
     inconsistent behavior that results from it.
   - `true`
+* - `hive.metastore-cache.cache-missing`
+  - Enable caching that a table is missing to prevent future metastore calls for
```

Comment:
maybe - wdyt @colebow ? its longer but also clearer .. at least in my opinion

```suggestion
  - Enable caching the fact that a table is missing to prevent future metastore calls for
```